functions/v2/http-logging/index.js

```diff
@@ -0,0 +1,39 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+// [START functions_structured_logging]
+const cloudLogging = require('@google-cloud/logging');
+const functions = require('@google-cloud/functions-framework');
+
+functions.http('structured_logging', async (req, res) => {
+  const logging = new cloudLogging.Logging();
+  const log = logging.logSync('Structured_logging project');
+  const text = 'Hello, world!';
+
+  // The metadata associated with the entry
+  // See: https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#logseverity
+  const metadata = {
+    severity: 'NOTICE',
+    component: 'arbitrary-property',
+    httpRequest: req,
+  };
+  // Prepares a log entry
+  const entry = log.entry(metadata, text);
+  log.write(entry);
+
+  res.status(200).send(entry);
```

Comment:
Instead of sending the entry as the response, let's just default something like "A log message was written"